### PR TITLE
[yezidi] Attempt to change name of language to Ezidi

### PR DESCRIPTION
**Pull request by LornaSIL**

### CODE REVIEW

File: release/y/yezidi/HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Yezidi Change History
 ====================
 
+1.0.1 (2024-10-16)
+----------------
+* Change language name to Ezidi
+* Remove ku-Yezi and just use kmr-Yezi
```
Comment (mcdurdin) [inline release/y/yezidi/HISTORY.md:7]: `kmr` will normalize to `ku`; the langtags.json entry shows `ku-Yezi-GE` as the appropriate minimal tag.

File: release/y/yezidi/HISTORY.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 Yezidi Change History
 ====================
 
+1.0.1 (2024-10-16)
+----------------
+* Change language name to Ezidi
+* Remove ku-Yezi and just use kmr-Yezi
+
```
Comment (mcdurdin) [inline release/y/yezidi/HISTORY.md:8]:
```suggestion
* Block unused keys from emitting English letters

```

File: release/y/yezidi/HISTORY.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
 Yezidi Change History
 ====================
 
+1.0.1 (2024-10-16)
+----------------
+* Change language name to Ezidi
```
Comment (mcdurdin) [inline release/y/yezidi/HISTORY.md:6]: You've changed the keyboard name from 'Ezidi keyboard' to 'Yezidi' as well. Is that intended?

**Review by mcdurdin — DISMISSED** [review]
I think this is okay, just noting that the history doesn't match the changes precisely.

**Comment by DavidLRowe** [comment]
PHP file has 'Eezidi Keyboard Help'. Is that intended (that is, 'Eezidi' vs 'Ezidi' as found elsewhere)?

**Pull request merged**